Clarify limitation of background scheduling (#631)
timescale/docs.timescale.com-content: using-timescaledb/limitations.md

hypertables. In addition, the following limitations apply specifically
to distributed hypertables:

> - Distributed scheduling of background work is not supported. Background jobs 
>   created on an access node are scheduled and executed on this access node 
>   without distributing the jobs to all data nodes of the distributed hypertable.

Reviewer: ```suggestion
- Distributed scheduling of background jobs is not supported. Background jobs 
  created on the access node are scheduled and executed on the access node 
  without distributing the jobs to any data nodes.
```
